ftp/src/main/scala/akka/stream/alpakka/ftp/impl/FtpBrowserGraphStage.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ftp
+package impl
+
+import akka.Done
+import akka.stream.stage.{ GraphStageWithMaterializedValue, OutHandler }
+import akka.stream.{ Attributes, Outlet, SourceShape }
+import scala.concurrent.{ Future, Promise }
+
+private[ftp] trait FtpBrowserGraphStage[FtpClient]
+    extends GraphStageWithMaterializedValue[SourceShape[FtpFile], Future[Done]] {
```

Comment:
Not really. The only purpose of Future[Done] is to check termination. Should I change it and not materialize?.